app/views/foreman_leapp/job_templates/leapp_preupgrade.erb

```diff
@@ -13,15 +13,20 @@ template_inputs:
   required: true
   default: "ga"
   options: "ga\ne4s\neus\naus"
+- name: Target
```

Comment:
```suggestion
- name: Target version
```
Target what? I'm assuming a minor version.
Can we rename it?

Reply:
I'm using Target as it is the name of the option used in the leapp command.

Reply:
I can see that. Does it mean we cannot rename it? It isn't a blocker for you PR.